Pyjion/absint.cpp

```diff
@@ -200,16 +209,27 @@ bool AbstractInterpreter::interpret() {
                 case NOP: break;
                 case ROT_TWO:
                 {
-                    auto tmp = lastState[lastState.stack_size() - 1];
-                    lastState[lastState.stack_size() - 1] = lastState[lastState.stack_size() - 2];
-                    lastState[lastState.stack_size() - 2] = tmp;
+                    // ROT_TWO currently assumes we have native ints on the stack,
+                    // so we force escape here...  We should fix that up in the future
+                    // and use pop_no_escape
+                    auto tmp = lastState.pop();
+                    auto second = lastState.pop();
+                    lastState.push(tmp);
+                    lastState.push(second);
+                    
+                    //auto tmp = lastState[lastState.stack_size() - 1];
```

Comment:
The commented out code is the ideal code (which is correct), we just need to update the code generator for what's now issue #88.  Once we do that this code can come back.